exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/Otel2PrometheusConverter.java

```diff
@@ -488,6 +490,14 @@ private Labels convertAttributes(
       if (scope.getVersion() != null) {
         labelNameToValue.putIfAbsent(OTEL_SCOPE_VERSION, scope.getVersion());
       }
+      String schemaUrl = scope.getSchemaUrl();
+      labelNameToValue.putIfAbsent(OTEL_SCOPE_SCHEMA_URL, schemaUrl == null ? "" : schemaUrl);
```

Comment:
Why not:
```suggestion
      if (schemaUrl != null) {
        labelNameToValue.putIfAbsent(OTEL_SCOPE_SCHEMA_URL, schemaUrl);
      }
```

Reply:
@ArthurSens I think you said that this value should be set even if null - can you point to the spec?